### DISCUSSION TITLE
Playlist tabs centered

### DISCUSSION THREAD

**Discussion by spamegg1** [DeaDBeeF-Player/deadbeef#3072] [Q&A] [answered]
Is it possible to make the Playlist tabs centered? I tried messing around in Design mode but could not find a way to do it.
![Screenshot from 2024-04-04 11-52-22](https://github.com/DeaDBeeF-Player/deadbeef/assets/4255997/50267536-825d-4afa-a39d-59cc1f9299b6)

**Comment by Oleksiy-Yakovenko** [comment, marked as answer]
Probably no existing way, nothing I could come up with.

**Reply by spamegg1** [reply]
Thanks!
